### PR TITLE
Add faker-eddy-malou third-party library

### PR DESCRIPTION
**Pull request by Metrakit**
Hi, I submit you a small PR for add my third-pirty package.

PS : thanks for your awesome conference in BlendWebMix about React components & legos

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1713,6 +1713,7 @@ echo $faker->VAT; //23456789
 * [bluemmb/Faker-PicsumPhotos](https://github.com/bluemmb/Faker-PicsumPhotos): Generate images using [picsum.photos](http://picsum.photos/)
 * [er1z/fakemock](https://github.com/er1z/fakemock): Generate mocks using class-configuration and detection via Faker's guesser and Symfony asserts
 * [xvladxtremal/Faker-LoremFlickr](https://github.com/xvladxtremal/Faker-LoremFlickr): Generate images using [loremflickr.com](http://loremflickr.com/)
+* [Metrakit/faker-eddy-malou](https://github.com/Metrakit/faker-eddy-malou): Generete French Eddy Malou sentences & paragraphs
```
Comment (localheinz) [inline readme.md]: @Metrakit 

Can you resolve conflicts and fix the typo (`generete` should be `generate`), please?

**Comment by Metrakit** [comment]
Ooops sorry for the typo. Done

File: readme.md (hunk 1 above)
Comment (Metrakit) [reply readme.md]: Ooops sorry for the typo. Done

**Comment by localheinz** [comment]
Thank you, @Metrakit and @pimjansen!

**Pull request merged**